Sources/SwiftBuildSupport/SwiftBuildSystem.swift

```diff
@@ -52,7 +52,14 @@ func withSession(
         _ diagnostics: [SwiftBuild.SwiftBuildMessage.DiagnosticInfo]
     ) async throws -> Void
 ) async throws {
-    switch await service.createSession(name: name, resourceSearchPaths: packageManagerResourcesDirectory.map { [$0.pathString] } ?? [], cachePath: nil, inferiorProductsPath: nil, environment: nil) {
+    switch await service.createSession(
+        name: name,
+        developerPath: nil,
```

Comment:
developerPath defaults to nil, so it shouldn't be needed here. Were you seeing the build fail in some configuration?

Reply:
I think my dependencies got in to a bad state, I switched to a known good toolchain, cleaned and rebuilt and its not an issue.